supersuit/utils/agent_indicator.py

```diff
@@ -74,13 +72,13 @@ def change_observation(obs, space, indicator_data):
         if ndims == 1:
             old_len = len(obs)
             new_obs = np.pad(obs, (0, num_indicators))
-            new_obs[indicator_num + old_len] = 1.0
+            new_obs[indicator_num + old_len] = np.max(space.high)
             return new_obs
         elif ndims == 3 or ndims == 2:
             obs = obs if ndims == 3 else np.expand_dims(obs, 2)
             old_shaped3 = obs.shape[2]
             new_obs = np.pad(obs, [(0, 0), (0, 0), (0, num_indicators)])
-            new_obs[:, :, old_shaped3 + indicator_num] = 1.0
+            new_obs[:, :, old_shaped3 + indicator_num] = np.max(space.high)
```

Comment:
What's the reason for this change as well?

Reply:
Same reason with https://github.com/Farama-Foundation/SuperSuit/pull/142#discussion_r865863503